packages/react-components/react-nav-preview/src/components/NavDrawer/renderNavDrawer.tsx

```diff
@@ -1,22 +1,11 @@
 /** @jsxRuntime automatic */
 /** @jsxImportSource @fluentui/react-jsx-runtime */
 
-import { assertSlots } from '@fluentui/react-utilities';
-import type { NavDrawerState } from './NavDrawer.types';
+import { DrawerState, renderDrawer_unstable } from '@fluentui/react-drawer';
```

Comment:
I think the state also needs to be splitted into type import?
```suggestion
import { renderDrawer_unstable } from '@fluentui/react-drawer';
import type { DrawerState } from '@fluentui/react-drawer';
```